dts/common/nordic/nrf54h20.dtsi

```diff
@@ -629,6 +629,7 @@
 				interrupts = <228 NRF_DEFAULT_IRQ_PRIORITY>;
 				power-domains = <&gpd NRF_GPD_FAST_ACTIVE1>;
 				#pwm-cells = <3>;
+				clocks = <&hsfll120>;
```

Comment:
Could you place it right after `interrupts` for consistency with other peripherals?